tests/test_resource.py

```diff
@@ -57,12 +58,7 @@ def setup_method(self):
     def test___call__wrong_uri(self):
         resp = self.res()
         assert isinstance(resp, Response)
-        assert resp.to_dict() == {
-            "body": '{"message":"Server is not able to produce a response"}',
-            "headers": {},
-            "statusCode": 421,
-            "isBase64Encoded": False,
-        }
+        assert resp.status_code == HTTPStatus.MISDIRECTED_REQUEST
```

Comment:
Shouldn't this be just 404? This 421 code doesn't even have a page at MDN, I'd be really scratching my head if I saw it somewhere